routes/api.js

```diff
@@ -4,6 +4,11 @@ var authentication = require('../authentication');
 var request = require('request-promise');
 var url = require('url');
 
+var  apiUrl = 'https://cad.onshape.com';
```

Comment:
Two spaces between var and apiUrl, does this repo have a linter?

Reply:
No - no lint tools, but I'll fix this. Thanks for the sharp eyes!